info.json

```diff
@@ -1,9 +1,12 @@
 {
     "name": "circuit-color-lamp-light-parity",
-    "version": "0.1.0",
+    "version": "0.1.1",
     "title": "Colored Lamp Normalizer",
     "author": "asher_sky",
     "description": "Modifies circuit-controlled colored lamps to shine with the same radius and intensity as manually-colored ones",
     "factorio_version": "2.0",
-    "dependencies": ["base >= 2.0"]
+    "dependencies": ["base >= 2.0"],
+    "package": {
+        "ignore": ["**/*.psd"]
+    }
```

Comment:
well, I want the .psd file to remain in the repository so I can use it to make new versions of the thumbnail if needed, but I don't want to distribute it in the releases because it adds a lot of file size to the zip